docs/core/install/remove-runtime-sdk-versions.md

```diff
@@ -0,0 +1,132 @@
+author: thraka
+ms.author: adegeo
+ms.date: 04/28/2020
+zone_pivot_groups: operating-systems-set-one
+---
+
+# How to remove the .NET Core Runtime and SDK
+
+Over time, as you install updated versions of the .NET Core runtime and SDK, you may want to remove outdated versions of .NET Core from your machine. Removing older versions of the runtime may change the runtime chosen to run shared framework applications, as detailed in the article on [.NET Core version selection](../versions/selection.md).
+
+## Should I remove a version?
+
+The [.NET Core version selection](../versions/selection.md) behaviors and the runtime compatibility of .NET Core across updates enables safe removal of previous versions. .NET Core runtime updates are compatible within a major version 'band' such as 1.x and 2.x. Additionally, newer releases of the .NET Core SDK generally maintain the ability to build applications that target previous versions of the runtime in a compatible manner.
+
+In general, you only need the latest SDK and latest patch version of the runtimes required for your application. Instances where keeping older SDK or Runtime versions include maintaining *project.json*-based applications. Unless your application has specific reasons for earlier SDKs or runtimes, you may safely remove older versions.
+
+## Determine what is installed
+
+Starting with .NET Core 2.1, the .NET CLI has options you can use to list the versions of the SDK and runtime that are installed on your machine.  Use [`dotnet --list-sdks`](../tools/dotnet.md#options) to see the list of SDKs installed on your machine. Use [`dotnet --list-runtimes`](../tools/dotnet.md#options) to see the list of runtimes installed on your machine. For more information, see [How to check that .NET Core is already installed](how-to-detect-installed-versions.md).
+
+## Uninstall .NET Core
+
+::: zone pivot="os-windows"
+
+.NET Core uses the Windows **Apps & features** dialog to remove versions of the .NET Core runtime and SDK. The following figure shows the **Apps & features** dialog. You can search for **core sdk** to filter and show installed versions of .NET Core.
+
+![Add / Remove programs to remove .NET Core](./media/remove-runtime-sdk-versions/programs-and-features.png)
+
+Select any versions you want to remove from your machine and click **Uninstall**.
+
+::: zone-end
+
+::: zone pivot="os-linux"
+
+There are more options to uninstall .NET Core (either SDK or runtime) on Linux. The best way for you to uninstall .NET Core is to mirror the action you used to install .NET Core. The specifics depend on your chosen distribution and the installation method.
+
+> [!IMPORTANT]
+> For Red Hat installations, consult the [Red Hat Getting Started Guide](https://access.redhat.com/documentation/en-us/net_core/2.0/html/getting_started_guide/gs_install_dotnet#install_register_rehel) for information on installing and uninstalling .NET Core.
```

Comment:
it's not a MS link, but will it work if you drop the /en-us?

Reply:
No, it doesn't work. It seems on the language dropdown on the page, only English is available.